runtime/shared_common/shrinit.cpp

```diff
@@ -80,6 +80,7 @@ extern "C" {
 
 
 #define SHRINIT_NAMEBUF_SIZE 256
+#define SHRINIT_LOW_FREE_DISK_SIZE (1024 * 1024 * 1024)
```

Comment:
Lets pick something bigger, say 3GB - 6GB, so the shared cache is only taking 10% - 5% of the available space.